core/src/main/java/io/aiven/klaw/model/Overview.java

```diff
@@ -0,0 +1,11 @@
+package io.aiven.klaw.model;
+
+import lombok.Data;
+
+@Data
+public class Overview {
```

Comment:
class name looks vague, may be TopicOverviewAttributes or anything else

Reply:
ResourceOverviewAttribitues as its the base for Topics and schemas and possible connectors in the future?